app/assets/stylesheets/items.css

```diff
@@ -1,6 +1,11 @@
+.new-ticket-form .item-form-ticket {
+  display: none;
+}
+
```

Comment:
so does this just make it only send the form info when you click submit if the field is not hidden?